common/server_room.cpp

```diff
@@ -232,6 +234,18 @@ void Server_Room::say(const QString &userName, const QString &s, bool sendToIsl)
     event.set_name(userName.toStdString());
     event.set_message(s.toStdString());
     sendRoomEvent(prepareRoomEvent(event), sendToIsl);
+
```

Comment:
If we did that, there would never be any chat history added would there not be?  Or an I miss-interpreting the line note?  If it always starts at 0 , then the block of code would be skipped all the time.